### ISSUE TITLE
[FEATURE] Web support

### ISSUE THREAD

**Issue by kawa89** [jhomlala/betterplayer#142]
Great library, I've migrated my [tweet_ui](https://pub.dev/packages/tweet_ui) library from `chewie` to `betterplayer` ;-) !
Are you planning to add web support to it?

**Comment by jhomlala** [comment]
Thank you for using Better Player!
At this moment, I have no plans for implementing Better Player on web. There are many reasons why:
* Flutter Web is not stable at this moment
* Better Player is still in development and API is not stable at it's subject of change
* Adding support for third platform increments amount of work that needs to be done, but unfortunately my time is very limited

Anyway, Better Player is open source, so if you've time to do web implementation, feel free to create PR. I always appreciate community work!

**Comment by JohnGalt1717** [comment]
@jhomlala Could we get platform interface stuff going so that other platforms can be easily added?  Now that Web is about to  go final in a few weeks and the desktop platforms into beta, (maybe not windows) it would be nice to have it all structured so adding the other platforms is straightforward for the person doing the implementing (I.e. I'd probably implement web and windows uwp once this is done)

**Comment by ekuusi** [comment]
I second @JohnGalt1717 's request. Better Player will not work on Flutter-Pi due to BP using its own `video_player_platform_interface` instead of the one by `video_player`. 

Therefore https://pub.dev/packages/omxplayer_video_player will not work with BP, whereas it works fine with Chewie.
